### PR TITLE
Allow report to render for Aquarius-only training sites by removing

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
requirement that stationName be present in json.

### CODE REVIEW

File: R/extremes-data.R [hunk 1]
```diff
@@ -7,6 +7,8 @@
 extremesTable <- function(rawData){
   
   data <- applyQualifiers(rawData)
+  l_qual <- data$dv$qualifiers
+  if (length(l_qual)==0) return ("The dataset requested is empty.")
```
Comment (thongsav-usgs) [inline R/extremes-data.R:11]: The data does not have to have qualifiers, this is cutting off rendering of valid data sets.

File: inst/extremes/extremes.Rmd [hunk 2]
```diff
@@ -26,5 +26,8 @@ cat(getLogo())
 tbl <- extremesTable(data)
 # formTable <- padTable(tbl)
 # cat(formTable)
-kable(tbl, align='r', row.names=FALSE)
+if(!identical(tbl, "The dataset requested is empty.")){
```
Comment (thongsav-usgs) [inline inst/extremes/extremes.Rmd:29]: I would nest this boolean logic a function. Right now you are relying on two hardcoded strings from two different places to be equal. A dev may change it in one place but not the other.

Comment (kmschoep-usgs) [inline inst/extremes/extremes.Rmd:29]: this logic is all over repgen.  I can change it here, but we should put in a tech debt task to apply the change everywhere.

Comment (thongsav-usgs) [inline inst/extremes/extremes.Rmd:29]: Yes, there are a lot of instances where we need to consolidate shared logic, that's why utils-shared was created.

File: R/extremes-data.R (hunk 1 above)
Comment (kmschoep-usgs) [inline R/extremes-data.R:11]: I'll need an example data set with no qualifiers but other valid data.

**Pull request merged**